buildSrc/src/main/kotlin/Publishing.kt

```diff
@@ -4,22 +4,26 @@ import org.gradle.api.publish.maven.MavenPublication
 import org.gradle.kotlin.dsl.create
 import org.gradle.kotlin.dsl.extra
 import org.gradle.kotlin.dsl.get
+import java.io.FileInputStream
 import java.nio.file.Paths
+import java.util.Properties
 
 object Publishing {
-    const val versionName = "3.0.4"
+    const val versionName = "3.0.5"
 
-    const val url = "https://github.com/trbnb/MvvmBase"
-    const val gitUrl = "https://github.com/trbnb/MvvmBase.git"
-    const val licenseUrl = "https://github.com/trbnb/MvvmBase/blob/master/LICENSE"
-    const val groupId = "de.trbnb"
+    const val url = "https://github.com/appmotion-gmbh/MvvmBase"
+    const val gitUrl = "https://github.com/appmotion-gmbh/MvvmBase.git"
+    const val licenseUrl = "https://github.com/appmotion-gmbh/MvvmBase/blob/master/LICENSE"
+    const val groupId = "io.github.appmotion-gmbh"
```

Comment:
Maybe we can also extract all this to Gradle Properties, so we remain compatible with the base repo.